reduction_scripts/reduce_data.py

```diff
@@ -779,91 +882,103 @@ def run_save_3dcube(metadata, prev_suffix, curr_suffix, **args):
     # Classify all raw data (red and blue arm)
     obs_metadatas = classify(data_dir, naxis2_to_process)
 
-
     for arm in obs_metadatas.keys():
-        #------------------------------------------------------------------------
-        #      LOAD JSON FILE WITH USER REDUCTION SETUP             
-        #------------------------------------------------------------------------
+        # ------------------------------------------------------------------------
+        #      LOAD JSON FILE WITH USER REDUCTION SETUP
+        # ------------------------------------------------------------------------
         obs_metadata = obs_metadatas[arm]
         reduction_scripts_dir = os.path.dirname(__file__)
         working_dir = os.getcwd()
 
```

Comment:
Maybe is a good idea to put `reduction_scripts_dir` and `working_dir` before the loop?

Reply:
done!